Prace_domowe/Praca_domowa2/adrianna_grudzien/pd2.ipynb

```diff
@@ -0,0 +1,1927 @@
+{
```

Comment:
<p>To dobrze czy źle?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>